macros/src/lib.rs

```diff
@@ -7,7 +7,9 @@ mod command;
 mod modal;
 mod util;
 
+use darling::Error;
```

Comment:
Probably better to refer to this using the path when used instead of importing like this, as there are a couple error types in play in this file.